pkg/controllers/config/labels.go

```diff
@@ -14,6 +14,14 @@
 
 package config
 
+const (
+	DefaultCloudResourcePrefix = "nephe"
+)
+
+type ControllerConfig struct {
```

Comment:
why is the struct here..? Its mostly for const

Reply:
Because controller config might contains more properties. This is how they followed in antrea controller.

Reply:
I think we can have config.go and move ControllerConfig struct and ConfigMapName/Namespace constants there.